packages/manager/modules/carbon-calculator/src/dashboard/routing.js

```diff
@@ -8,6 +10,12 @@ export default /* @ngInject */ ($stateProvider) => {
       dashboardLink: /* @ngInject */ ($state) => $state.href('app.dashboard'),
       breadcrumb: /* @ngInject */ ($translate) =>
         $translate.instant('carbon_calculator_my_carbon_footprint'),
+      trackClick: /* @ngInject */ (atInternet) => {
```

Comment:
Are you using this in controller or html ?

Reply:
none of them. Actually, tracking data are sent when page is displayed. Here tracking is done just before displaying the page.
I can move this part in the `controller.$onInit` if you prefer  ?